dks-host/dks

```diff
@@ -0,0 +1,105 @@
+#!/bin/sh
```

Comment:
systemd unit files are generally much easier to read & write, although I must admit to having very little experience with them myself. May be worth a future ticket to explore replacing this old-style init script with a systemd unit file. Or perhaps see if Mark/Steve have any experience to replace this now?

Reply:
We're following the pattern of Kafka broker here. Happy to explore systemd option in a future ticket but the current pattern will give us a quick win - what do you think?